docs/hsi.xml

```diff
@@ -0,0 +1,875 @@
+      <title>Runtime Behavior</title>
+      <para>
+        Orthogonal to the security features provided by the firmware there are
+        other security considerations related to the firmware which may require
+        internet access to discover or that runtime OS changes directly affect
+        the security of the firmware.
+        It would not make sense to have <emphasis>“have updates on the LVFS”</emphasis>
+        as a requirement for a specific security level as this would mean
+        offline the platform might be a higher level initially but as soon as
+        it is brought online it is downgraded which would be really confusing to
+        users.
+        The “core” security level will not change at runtime, but the suffix may.
+      </para>
+    </refsect2>
+
+    <refsect2 id="hsi-level0">
+      <title>HSI:0 (Insecure)</title>
+      <para>
+        The lowest security level with very few detected firmware protections.
+        This is the default security level if no tests can be run or some tests
+        in the next security level have failed.
+      </para>
+    </refsect2>
+
+    <refsect2 id="hsi-level1">
+      <title>HSI:1 (Critical)</title>
+      <para>
+        This security level corresponds to the most basic of security protections
+        considered essential by security professionals.
+        Any failures at this level would have critical security impact.
+      </para>
+    </refsect2>
+
+    <refsect2 id="hsi-level2">
+      <title>HSI:3 (Theoretical)</title>
+      <para>
+        This security level corresponds to firmware security issues that pose a
+        theoretical concern or where any exploit would be difficult or
+        impractical to use.
+        At this level various technologies may be employed to protect the boot
+        process from modification by an attacker with local access to the machine.
+      </para>
+    </refsect2>
+
+    <refsect2 id="hsi-level4">
+      <title>HSI:4 (System Protection)</title>
+      <para>
+        This security level corresponds to out-of-band protection of the system
+        firmware perhaps including recovery.
+      </para>
+    </refsect2>
+
+    <refsect2 id="hsi-level5">
+      <title>HSI:5 (System Attestation)</title>
+      <para>
+        This security level corresponds to out-of-band attestation of the system
+        firmware.
+        There are currently no tests implemented for HSI:5 and so this security
+        level cannot yet be obtained.
+      </para>
+    </refsect2>
+
+    <refsect3 id="runtime-u">
+      <title>HSI Runtime Suffix <code>U</code></title>
+      <para>
+        Updates available on the <ulink url="https://fwupd.org/">
+        Linux Vendor Firmware Service </ulink> which are less than 12 months old.
+      </para>
+    </refsect3>
+
+    <refsect3 id="runtime-a">
+      <title>HSI Runtime Suffix <code>A</code></title>
+      <para>
+        Attestation data is available to verify the current PCR0 TPM hash.
```

Comment:
Something about this being UFEI specific and linking to an explanation of PCR0 would be useful here.

Reply:
Also this is the first time a TPM is mentioned and it might need to be explained too.

Reply:
How about

        Attestation data is available to verify the current system firmware.
        For most UEFI platforms, this is usually the Trusted Platform Module (TPM)
        PCR0 hash, but other attestation checksums could be used.

Reply:
Sounds good, and it's good to disambiguate that something non-UEFI may exist some day.